src/pages/index.js

```diff
@@ -205,3 +355,60 @@ formSelectors.forEach((selector) => {
 });
 
 // use in open method
+
+/* -------------------------------------------------------------------------- */
+/*                               API CODE BELOW                               */
+/* -------------------------------------------------------------------------- */
+
+//delete is hard coded for now, will eventgually add input
+//api.deleteCard("/cards/", "652e8963beb738001aecbb32");
+
+//api.likeCard("/cards/", "65306057369eff001aa738d9");
+//api.disLikeCard("/cards/", "65306057369eff001aa738d9");
+/* -------------------------------------------------------------------------- */
+/*                               example request                              */
+/* -------------------------------------------------------------------------- */
+
+// function basicCall() {
+//   return fetch("https://around-api.en.tripleten-services.com/v1/cards", {
+//     method: "GET",
+//     headers: {
+//       authorization: "ed64b8cb-b7cb-483b-9267-3e840bed2c98",
+//     },
+//   })
+//     .then((res) => {
+//       if (res.ok) {
+//         return res.json();
+//       }
+//       return Promise.reject(`Something went wrong ${res.status}`);
+//     })
+//     .then((result) => {
+//       console.log(result);
+//     })
+//     .catch((err) => console.error(err))
+//     .finally(console.log("Zug Zug"));
+// }
+
+// basicCall();
+
+// fetch("https://around-api.en.tripleten-services.com/v1", {
+//   method: "POST",
+//   body: JSON.stringify({
+//     name: "Sam",
+//   }),
+//   headers: {
+//     "Content-Type": "application/json; charset=UTF-8",
+//   },
+// })
+//   .then((response) => response.json())
```

Comment:
Needs correction: I don't see the loading state on the avatar form button

Reply:
Had a typo fixed now :(